mem.go

```diff
@@ -2,12 +2,61 @@ package sqrl
 
 import (
 	"fmt"
+	"math/big"
 
-	"github.com/shirou/gopsutil/mem"
+	"github.com/dmmcquay/likexian/host-stat-go"
+	"github.com/dmmcquay/sqrl/dustin/go-humanize"
 )
 
-func MemInfo() {
+func meminfo() (uint64, uint64) {
 	v, _ := mem.VirtualMemory()
+	var total uint64
+	total = v.Total
+	free := v.Free
+	return total, free
+}
+
+func swapinfo() (uint64, uint64) {
+	s, _ := mem.SwapMemory()
+	total := s.Total
+	free := s.Free
+	return total, free
+}
 
-	fmt.Println(v)
+func versioninfo() (string, string, string) {
+	i, _ := host_stat.GetHostInfo()
+	osVersion := i.OSType
+	hostname := i.HostName
+	version := i.OSRelease
+	return osVersion, hostname, version
+}
+func mem() {
```

Comment:
I totally understand what you are trying to do here and for the first shot, it is a great attempt.  Let me give you some ideas of how I want you to move forward.

Lets go ahead and define a struct that encompasses all the data you need to get (well, two actually).  

``` 
type Ram struct {
    total, free, swapSpace uint64
}
```

```
type OS struct {
    hostname, kernel, version string
} 
```
Create me a function that I can call and it will return me a OS struct and a Ram struct.

Hopefully that makes sense...